app/src/main/java/io/github/droidkaigi/confsched2017/util/AssetsUtil.java

```diff
@@ -23,15 +23,23 @@ private AssetsUtil() {
      */
     public static String loadJSONFromAsset(Context context, final String jsonFileName) {
         String json = null;
+        InputStream is = null;
         try {
-            InputStream is = context.getAssets().open("json/" + jsonFileName);
+            is = context.getAssets().open("json/" + jsonFileName);
             int size = is.available();
             byte[] buffer = new byte[size];
             is.read(buffer);
-            is.close();
             json = new String(buffer, "UTF-8");
         } catch (IOException e) {
             Timber.e(AssetsUtil.class.getSimpleName(), e.getMessage(), e);
+
+            try {
+                if (is != null) {
+                    is.close();
+                }
+            } catch (IOException exp) {
+                Timber.e(AssetsUtil.class.getSimpleName(), e.getMessage(), exp);
+            }
         }
```

Comment:
input stream is not closed when succeeded.
As @sys1yagi said, closing stream in `finally` block works well in this case.